run_app.sh

```diff
@@ -16,7 +16,16 @@ if [ -z "$PORT" ];
   then PORT=8080
 fi
 
-prometheus_multiproc_dir=$TEMPDIR gunicorn wsgi -w $NUM_WORKERS --threads $THREADS -b 0.0.0.0:$PORT --log-level=$LOG_LEVEL --access-logfile=- --config ./gunicorn.conf.py
+if [ -z "$SERVICE_MODE" ];
+  then SERVICE_MODE=REST
```

Comment:
Sorry for the churn, would you consider `REST_API` as REST alone isn't obvious to me that is the API its spinning up.